### DISCUSSION TITLE
Issues since Jan with mp4 and m4a files

### DISCUSSION THREAD

**Discussion by ThE-iTCH** [aclap-dev/video-downloadhelper#224] [Help and Q&A] [answered]
### Your question

Since January it's not detecting m4a or mp4 files from a website i used all the time. Other tools work on the same site. Did something change? I even tried using the coapp with no change

### Addon & CoApp version and details

version: 8.2.1.1
channel: stable
build date: 2024-02-19
build options: linuxlic: false, noyt: false, browser: firefox
lang: en-US
license: unset
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:109.0) Gecko/20100101 Firefox/115.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\SMPTADMIN",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f360955"
}

**Comment by jcfrog** [comment]
Hi,
Can you share a public link as an example?

**Reply by ThE-iTCH** [reply]
https://www.multitracks.com/ anything from here and it used to work

Specific link for an item
https://www.multitracks.com/songs/Hillsong-United/People/Echoes-(Till-We-See-the-Other-Side)/rehearsalmix/?rmixID=23772589&key=1

**Reply by jcfrog** [reply]
hum, sorry but these links are not public.

**Reply by ThE-iTCH** [reply]
Please excuse me - Here's one I checked works on a different browser - it's just the sample version, but should aid in checking it

https://www.multitracks.com/songs/Elevation-Worship/CAN-YOU-IMAGINE-/Praise/

**Reply by paulrouget** [reply]
Right. It only shows videos, no audio source.

I'll investigate asap! Thanks for the report.

**Reply by ThE-iTCH** [reply]
Thanks!

**Reply by paulrouget** [reply, marked as answer]
Alright - I found out the issue. This will be fixed in upcoming 8.2.1.4 release.

**Reply by ThE-iTCH** [reply]
Thanks Appreciate it! that's great news
